### ISSUE TITLE
Inconsistent note-dot behaviour across note-off triggers

### ISSUE THREAD

**Issue by 0xHexaDecimator** [schismtracker/schismtracker#413]
When a note-off is triggered by ===, the note-dot disappears immediately
When a note-off is triggered by NNA (or instrument effect S71), the note-dot fades linked to velocity in IT or volume by default in Schism. I believe this is the expected behaviour.
Volume and velocity bars fade as expected regardless of how the note-off is triggered
https://github.com/schismtracker/schismtracker/assets/151794548/f0e272e6-fbab-43f6-9458-a0bf1feb2eb7
[note-dot_behaviour.zip](https://github.com/schismtracker/schismtracker/files/14972377/note-dot_behaviour.zip)
This also occurs with note-fades (~~~)
https://github.com/schismtracker/schismtracker/assets/151794548/60f9885f-367a-4769-86ea-dc9c860403a2

**Comment by 0xHexaDecimator** [comment]
In checking past versions, note-dots appear to behave properly with Schism 20221201 in the scenario above.

**Comment by mrpapersonic** [comment]
https://github.com/schismtracker/schismtracker/blob/ddc2e86f860e5c1e2253919d4c13a19b36b3d75f/player/effects.c#L1434

I believe it's this line that's overwriting the real note value, which is what the note dot code looks at.

**Comment by mrpapersonic** [comment]
The note dots look to be working correctly now but they need some more testing for me to close this

**Comment by mrpapersonic** [comment]
They seem to be working fine, however they are based on the current actual output of the channel rather than what the volume multiplier is (which IMO is a good change anyway, but I don't know the IT behavior)

**Comment by 0xHexaDecimator** [comment]
> They seem to be working fine

Yeah, they work well enough now, though sometimes a dot gets stuck at lowest volume after a note has ended (no NNAs active)

<img width="441" height="119" alt="Image" src="https://github.com/user-attachments/assets/f19e3651-c546-4735-8d8e-977ea4be92d4" />
(screenshot from a 4 channel .mod)

> however they are based on the current actual output of the channel rather than what the volume multiplier is (which IMO is a good change anyway, but I don't know the IT behavior)

The VU meter update helped that a ton, as long as the meter isn't flickering through a held note it works well. IT's behaviour is basically what you see in Classic Mode's note dots.
